### PR TITLE
Feature/team logos

### PR DESCRIPTION
**Pull request by NickEwer**
- Added team logos as optional scoreboard item
- Added team logo prompt to consoleapp series
- consoleapp takes a string URL

- CSS/HTML refactor to accommodate logo

In game with logos
image

In game without logos
image

Console app
image

### CODE REVIEW

File: overlay-app/src/css/Scoreboard.css [hunk 1]
```diff
@@ -1,82 +1,144 @@
-.scoreboard {
+.scoreboard-wrapper {
```
Comment (NickEwer) [inline overlay-app/src/css/Scoreboard.css:1]: Added a wrapper here to fill the width of the screen, so I could `margin: 0 auto` to center the scoreboard no matter the scoreboard width

File: overlay-app/src/css/Scoreboard.css [hunk 2]
```diff
@@ -1,82 +1,144 @@
...
+  
+  --center-w: calc(var(--sw) / 10);
+  --center-hw: calc(var(--center-w) / 2);
+  --score-w: calc(var(--sw) * .04);
+	--tally-h: calc(var(--sw) * .0078);
+
+  width: var(--scoreboard-w);
+}
+
+.scoreboard.has-logos {
```
Comment (NickEwer) [inline overlay-app/src/css/Scoreboard.css:30]: Made the scoreboard bigger when logos are present so the space for the rest wasn't compromised

File: overlay-app/src/css/Scoreboard.css [hunk 3]
```diff
@@ -1,82 +1,144 @@
...
+	background-repeat: no-repeat;
+	background-size: contain;
+}
+
+.scoreboard .side {
+  float: left;
+  position: relative;
+  width: calc(var(--scoreboard-hw) - calc(var(--center-hw)));
+  height: var(--scoreboard-h);
+}
```
Comment (NickEwer) [inline overlay-app/src/css/Scoreboard.css:71]: Moved some of the CSS for shared things to a generic class, specifying exceptions below

File: overlay-app/src/components/Scoreboard.tsx [hunk 4]
```diff
@@ -74,41 +77,53 @@ export const Scoreboard: FC<ReturnType<typeof getState>> = (props) => {
...
   [rightTeamName, rightFontSize] = scaleText(rightTeamName, sizes);
+  let hasBothLogos = leftTeamLogo && rightTeamLogo;
 
   return (
-    <div className="scoreboard">
-      <div className="left">
-        <div className="name" style={{fontSize:leftFontSize}}>{leftTeamName}</div>
-        <div className="score">{teams[0].score}</div>
-      </div>
+    <div className="scoreboard-wrapper">
```
Comment (NickEwer) [inline overlay-app/src/components/Scoreboard.tsx:92]: Moved the tally score pieces into the left/right side to make the sizes of things relative to their parent container and rely less on the css variables, which could be more complicated when tweaking the sizes of things

**Review by tynidev — APPROVED** [review]

**Pull request merged**